src/geoserver-publish-imagemosaic/index.js

```diff
@@ -48,9 +57,21 @@ const geoserverPublishImageMosaic = async (workerJob, inputs) => {
       // cf. grc.datastores.createImageMosaicStore
       throw 'Datastore does not exist.'
     }
+
+    // move coverage to coverage store folder to keep file structure clean in data directory
+    const fileName = path.basename(coverageToAdd);
+    const oldPath = coverageToAdd;
+    // the internal geoserver url always starts with 'file:', so we split it and use the second index
```

Comment:
usually file pathes start with three slashes, like this `file:///tmp/test.file`, so this may not always be successful. But if your tests look good i think its ok